apps/web/src/features/subjects/pages/SubjectGraphPage.tsx

```diff
@@ -40,16 +44,64 @@ export const SubjectGraphPage = () => {
 
   const lines = useGraphLines({ selectedMeasures });
 
+  const graphRef = useRef<HTMLDivElement>(null);
+
   const handleSelectForm = (id: string) => {
     setInstrumentId(id);
     setSelectedMeasures([]);
   };
 
+  const handleGraphDownload = async () => {
+    if (!graphRef.current) return;
+
+    const selectGraph = graphRef.current;
+
+    const canvas = await html2canvas(selectGraph, {
+      onclone: (_, element) => {
+        const graphDesc = document.createElement('div');
```

Comment:
This should probably be a `p` element, or else a `p` within a `div`. It is a somewhat bad practice to insert text directly into a `div`, since it can lead to accessibility issues.

Reply:
Replaced generated divs with p tags instead.